client/src/Poem.jsx

```diff
@@ -15,6 +16,7 @@ const Poem = ({ poem }) => {
     const NBSP = '\u00A0'; // Unicode value for non-breaking space
     return <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={index} />;
   });
+  const poemCommentsURL = '/api/comments/'.concat(poem.id);
```

Comment:
Used a template literal [here](https://github.com/michelmansour/virgil/pull/6/commits/d8229fb86c1dde13c9b2114e9751ce7852c19459#diff-27f2e40ff8e567cf38d1ee19c588501fR28) instead of disabling the linter.